packages/gasket-core/package.json

```
@@ -2,12 +2,19 @@
   "name": "@gasket/core",
   "version": "7.0.0-next.21",
   "description": "Entry point to setting up Gasket instances",
-  "main": "lib/index.js",
+  "module": "lib/index.js",
+  "main": "cjs/lib/index.js",
+  "exports": {
+    "import": "./lib/index.js",
+    "require": "./cjs/lib/index.js"
+  },
   "types": "lib/index.d.ts",
   "files": [
     "lib"
   ],
   "scripts": {
+    "clean": "rimraf cjs",
+    "build": "npm run clean && swc lib -d cjs",
```

Another thing I just noticed, you can remove the extra clean script and rimraf dep. Instead do:
```suggestion
    "build": "swc lib -d cjs --delete-dir-on-start",
```

---

Looks like using `--strip-leading-paths` is what we want to avoid the nested `lib` dir